rafthttp/http.go

```diff
@@ -117,11 +130,26 @@ func (h *streamHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	fromStr := strings.TrimPrefix(r.URL.Path, RaftStreamPrefix+"/")
+	var t streamType
+	switch path.Dir(r.URL.Path) {
+	case path.Join(RaftStreamPrefix, string(streamTypeMsgApp)):
```

Comment:
Could you illustrate this?
i think strings.contains may miss some case if substring appears in other part in the string.